src/schematics/ng-add.jasmine.ts

```diff
@@ -368,7 +369,7 @@ describe('ng-add', () => {
     });
 
     it('generates new files if starting from scratch', async () => {
-      const result = await setupProject(tree, {
```

Comment:
if `await` not needed then you also have to remove `async`, also applies this to the rest where `await` is removed